include/seqan/bam_io/bam_index_bai.h

```diff
@@ -61,6 +61,7 @@
 #ifndef INCLUDE_SEQAN_BAM_IO_BAM_INDEX_BAI_H_
 #define INCLUDE_SEQAN_BAM_IO_BAM_INDEX_BAI_H_
 
+
```

Comment:
:nail_care:

Reply:
The file is clearly unreadable without that newline 😃